### ISSUE TITLE
Document not cleared if it only has an automatically inserted \version

### ISSUE THREAD

**Issue by jeanas** [wbsoft/frescobaldi#1564]
Steps to reproduce:

- Make sure that in Edit > Preferences > General Preferences > New Document, "Create an empty document" is selected.
- When you open Frescobaldi, you have one tab, with an empty document. Now open any existing LilyPond file. The file is opened in the same tab, replacing the empty document that was there before. (On the other hand, if you insert some code into the document before opening a new one, you get a new tab).
- In the preferences, change the option to “Create a document that contains the LilyPond version statement”.
- Close and reopen Frescobaldi. You get a document with only a `\version` statement. Now open a LilyPond file. This time, it's opened in a new tab and you have to close the existing tab.

In other words, we currently use "is empty" as the criterion for replacing the current document, but it should be "has not been modified since it was created, perhaps with some prefilled content".
